app/data_definitions/from_cascade/two_column.xml

```diff
@@ -0,0 +1,731 @@
+                <radio-item value="Yes"/>
+                <radio-item value="No"/>
+            </text>
+            <group identifier="slides" label="Slides">
+                <group identifier="slide" label="Slide" multiple="true" maximum-number="10" minimum-number="1">
+                    <text identifier="header" label="Title"/>
+                    <text identifier="sub-title" label="subTitle"/>
+                    <text identifier="description" label="Description"/>
+                    <asset type="file" identifier="image" label="Image (1130x440)"/>
+                    <text identifier="altText" label="Image Alt-Text" required="true" help-text="Alternate text to describe image (required)"/>
+                    <text identifier="quote-author" label="Author Attribution" help-text="Will only display if the slide type is set to Quote"/>
+                    <text type="dropdown" identifier="text-background" label="Text Background" default="Transparent Black">
+                        <dropdown-item value="Transparent Black"/>
+                        <dropdown-item value="Solid Red"/>
+                    </text>
+                    <text type="dropdown" identifier="align" label="Slide Align" default="bottom-left" help-text="This will align the quote box relative to the masthead. This will only work with the slide type of quote.">
+                        <dropdown-item value="bottom-left"/>
+                        <dropdown-item value="bottom-right"/>
+                    </text>
+                    <group identifier="link" label="Link">
+                        <text identifier="link" label="External Link" help-text="full url (including http) to page outside of Cascade"/>
+                        <asset type="page" identifier="internalLink" label="Or Internal Link"/>
+                        <asset type="file" identifier="fileLink" label="Or File Link"/>
+                    </group>
+                </group>
+            </group>
+        </group>
+        <text type="radiobutton" identifier="showMasthead" label="Show Masthead" default="No" help-text="Becoming obsolete. Only here for legacy pages">
+            <radio-item value="Yes"/>
+            <radio-item value="No"/>
+        </text>
+        <text type="radiobutton" identifier="showImage" label="Custom Image" default="No" help-text="for OLD Masthead. Indicates to use Image below instead of default image">
+            <radio-item value="Yes"/>
+            <radio-item value="No"/>
+        </text>
+        <asset type="file" identifier="image" label="Image (1024x260)" help-text="for OLD Masthead. Large image to replace small default image"/>
+        <text identifier="altText" label="Image Alt-Text" help-text="Alternate text to describe image (required)"/>
+        <text identifier="description" label="Description" maxlength="150" help-text="for OLD Masthead. Visible caption under masthead image"/>
+    </group>
```

Comment:
Here's the full old masthead XSLT block:

```
<xsl:when test="showMasthead='Yes' and showImage='Yes'">
    <div class="masthead-old">
        <xsl:if test="link/label/text() != ''">
            <xsl:attribute name="class">masthead withLink</xsl:attribute>
        </xsl:if>
        <xsl:if test="showImage='Yes'">
            <xsl:if test="not(image/path='/')">
                <xsl:if test="not(altText='')">
                    <img alt="{altText}" src="{image/path}" width="1024"/>
                </xsl:if>
                <xsl:if test="altText=''">
                    <img alt="[photo description needed in old masthead]" src="{image/path}" width="1024"/>
                </xsl:if>
            </xsl:if>
        </xsl:if>
        <div>
            <xsl:value-of select="description"/>
            <xsl:if test="link/label/text() != ''">
                <a class="more">
                    <xsl:choose>
                        <xsl:when test="not(link/fileLink/path='/')">
                            <xsl:attribute name="href">
                                <xsl:value-of select="link/fileLink/path"/>
                            </xsl:attribute>
                        </xsl:when>
                        <xsl:when test="not(link/internalLink/path='/')">
                            <xsl:attribute name="href">
                                <xsl:value-of select="link/internalLink/path"/>
                            </xsl:attribute>
                        </xsl:when>
                        <xsl:otherwise>
                            <xsl:attribute name="href">
                                <xsl:value-of select="link/link"/>
                            </xsl:attribute>
                        </xsl:otherwise>
                    </xsl:choose>
                    <xsl:value-of select="link/label"/>
                </a>
            </xsl:if>
        </div>
    </div>
</xsl:when>
```

The same field is used for the more link in last block. 

I suspect that block doesn't exist and these tests always return false. Is that the case?